src/MonoDebugSession.cs

```diff
@@ -260,11 +256,18 @@ public override async void Launch(Response response, dynamic args)
 			var cmdLine = new List<String>();
 
 			bool debug = !getBool(args, "noDebug", false);
+			
 			if (debug) {
-				cmdLine.Add("--debug");
-				cmdLine.Add(String.Format("--debugger-agent=transport=dt_socket,server=y,address={0}:{1}", host, port));
+				if (!env.ContainsKey("MONO_ENV_OPTIONS"))  
+					env["MONO_ENV_OPTIONS"] = $" --debug --debugger-agent=transport=dt_socket,server=y,address={host}:{port}";
+				else
+					env["MONO_ENV_OPTIONS"] = env["MONO_ENV_OPTIONS"] + $" --debug --debugger-agent=transport=dt_socket,server=y,address={host}:{port}";
```

Comment:
I think we should append the user options at the end so that they override ours (e.g. if the user wants to pass `--debug=casts`)

```suggestion
				if (!env.ContainsKey("MONO_ENV_OPTIONS"))
					env["MONO_ENV_OPTIONS"] = $" --debug --debugger-agent=transport=dt_socket,server=y,address={host}:{port}";
				else
					env["MONO_ENV_OPTIONS"] = $" --debug --debugger-agent=transport=dt_socket,server=y,address={host}:{port} " + env["MONO_ENV_OPTIONS"];
```